third_party/googleapis/BUILD.bazel

```diff
@@ -24,7 +24,7 @@ exports_files(["LICENSE"])
 
 filegroup(
     name = "srcs",
-    srcs = glob(["**"]),
+    srcs = glob(["**"], allow_empty = True),
```

Comment:
@aiuto it does not look like this change got applied

Reply:
third_party has to be done in a separate PR from anything else.
Do things really fail without this?  That glob should never have anything empty.
If we can not do the change in third_party it would be nicer, because then we don't have to do a patch whenever we update in the future.

Reply:
I'll double check.

Reply:
@aiuto it seems that it was not needed in the end. I managed to get the CI green without this change https://github.com/bazelbuild/bazel/pull/15327
Because you worked on packaging, you might be interested in https://github.com/bazelbuild/bazel/pull/15327/files#diff-29ed1a1c7d731f96c7fe253b871566952b697712ea2d8d835f703b41dbbd13faR35

Reply:
Thanks.  That bit in tools/build_defs/pkg is irrelevant to anything. By Bazel 7.x it won't exist any more.